<launch>
  <arg
    name="model" />
  <param
    name="robot_description"
    textfile="$(find excavator1)/urdf/excavator1.urdf" />
  <node
    name="joint_state_publisher"
    pkg="excavator1"
    type="joint_state_publisher3.py" />
  <node
    name="robot_state_publisher"
    pkg="robot_state_publisher"
    type="robot_state_publisher" />
  <node
    name="rviz"
    pkg="rviz"
    type="rviz"
    args="-d $(find excavator1)/urdf.rviz" />
</launch>
